edx_event_bus_redis/internal/consumer.py

```diff
@@ -93,18 +93,24 @@ class RedisEventConsumer(EventBusConsumer):
         consumer_name: unique name for consumer within a group.
         last_read_msg_id: Start reading msgs from a specific redis msg id.
         check_backlog: flag to process all messages that were not read by this consumer group.
+        claim_msgs_older_than: claim pending msgs from other consumers in the group with idle time older
+        than a specific time (in milliseconds).
```

Comment:
If you run `make docs`, and look at the `<docs-root>/edx_event_bus_redis.internal.html#edx_event_bus_redis.internal.consumer.RedisEventConsumer.claim_msgs_older_than`, you will see:
![image](https://user-images.githubusercontent.com/30300520/232323015-51585cdd-933b-47d8-8b36-879191003d12.png)
To fix this, you need to indent the second line of the annotation:
```suggestion
        claim_msgs_older_than: claim pending msgs from other consumers in the group with idle time older
            than a specific time (in milliseconds).
```

Reply:
@Cup0fCoffee Thanks!!